web/src/main/java/org/cbioportal/proxy/ProxyController.java

```diff
@@ -43,20 +43,22 @@ public String proxy(@RequestBody(required = false) String body, HttpMethod metho
             String.class
         ).getBody();
     }
-
-    // TODO remove this endpoint after external projects switch to the new proxy
+    
     @RequestMapping("/oncokb/**")
     public String proxyOncokb(
         @RequestBody(required = false) String body,
         HttpMethod method,
         HttpServletRequest request
     ) throws URISyntaxException {
+        String token = request.getHeader("X-Proxy-User-Agreement");
```

Comment:
We can also use a different header name, but that requires adding the new header into the CORS allowed headers list.